### DISCUSSION TITLE
guild.members only returning bot member

### DISCUSSION THREAD

**Discussion by BlueNerf** [Rapptz/discord.py#10046] [Q&A]
### There is a discussion but it didn't have any responses so I made a new one.

I have this **code**:

`
@bot.command(name='members')
async def _members(ctx, guild_id: int):
  guild = bot.get_guild(guild_id)
  for member in guild.members:
    await ctx.send(member.display_name)
`
For some reason the line returns aren't working, sorry for that, it's my first time making a thing like this.
I would expect it to return the display_name of everyone in the server right?
### **_NO_**

yeah it only returns the **bot's** display name.
Could anyone help me with this?

**Comment by DA-344** [comment]
Make sure you have the members intent enabled and the guild is chunked.

And this _should_ be opened as an issue, not as a discussion.

**Reply by BlueNerf** [reply]
sorry, but what does it mean to have a guild chunked, and can you please give the docs for it or how I do it?

Thanks, BlueNerf

**Reply by DA-344** [reply]
https://discordpy.readthedocs.io/en/stable/api.html#discord.Guild.chunked
